types/node/dns.d.ts

```diff
@@ -292,7 +292,8 @@ declare module "dns" {
         resolveSrv: typeof resolveSrv;
         resolveTxt: typeof resolveTxt;
         reverse: typeof reverse;
-        cancel(): void;
+        setLocalAddress(ipv4?: string, ipv6?: string): void;
```

Comment:
set local address is not included in NodeJs 14. I think needs to wait until it's backported or till type definitions here are moved to v15 and v14 get their own folder like v12,...